config/gulp/file-upload.js

```diff
@@ -1,69 +1,51 @@
 require("dotenv").config();
-const { series, src } = require("gulp"),
-  vinylPaths = require("vinyl-paths"),
-  del = require("del"),
-  s3config = {
-    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
-    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+const gulp = require("gulp");
+const awspublish = require('gulp-awspublish');
+const del = require("del");
+const vinylPaths = require("vinyl-paths");
+
+// Create a new publisher using S3 options
+const publisher = awspublish.create({
+ //region: 'your-region-id', // Change to your AWS region
+  params: {
+    Bucket: 'digitalgov', // Change to your bucket name
   },
-  s3 = require("gulp-s3-upload")(s3config);
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+});
 
-/**
- * Uploads images in /processed folder to digitalgov s3 bucket
- * TODO: Refactor both functions into one
- * https://github.com/uswds/uswds-compile/blob/223fe67c335135c55cf1be9dead44e6d363e219d/gulpfile.js#L204-L232
- */
+// Define custom headers
+const headers = {
```

Comment:
Do we need these headers, what do they do?

Reply:
Removed and still works